.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# See: https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "build"
```

Comment:
We could standardize the prefixes as in `complyctl` and `complyscribe`: https://github.com/complytime/complyctl/blob/main/.github/dependabot.yml